### ISSUE TITLE
Solution folders seem not to be supported

### ISSUE THREAD

**Issue by Kralizek** [Velorien/DependencyGraphDrawer#1]
I tried to use the tool on a solution with several layers of solution folders, this is the output I got.

```txt
Unhandled exception: System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> System.IO.FileNotFoundException: Could not find file 'C:\Users\RenatoGolia\Development\InsightArchitectures\Varden\BookingPlatform\Services\engines'.
File name: 'C:\Users\RenatoGolia\Development\InsightArchitectures\Varden\BookingPlatform\Services\engines'
   at System.IO.FileSystem.GetAttributes(String fullPath)
   at System.IO.File.GetAttributes(String path)
   at DependencyGraphDrawer.GraphDrawer.<>c.<DrawDiagram>b__0_0(ProjectInSolution x) in C:\Code\DependencyGraphDrawer\DependencyGraphDrawer\GraphDrawer.cs:line 60
   at System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
   at DependencyGraphDrawer.GraphDrawer.DrawDiagram(GraphOptions options) in C:\Code\DependencyGraphDrawer\DependencyGraphDrawer\GraphDrawer.cs:line 65
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.Delegate.DynamicInvokeImpl(Object[] args)
   at System.Delegate.DynamicInvoke(Object[] args)
   at System.CommandLine.Invocation.ModelBindingCommandHandler.InvokeAsync(InvocationContext context)
   at System.CommandLine.Invocation.InvocationPipeline.<>c__DisplayClass4_0.<<BuildInvocationChain>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass23_0.<<UseParseErrorReporting>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass16_0.<<UseHelp>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass27_0.<<UseVersionOption>b__1>d.MoveNext()--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass25_0.<<UseTypoCorrections>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c.<<UseSuggestDirective>b__24_0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass22_0.<<UseParseDirective>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass11_0.<<UseDebugDirective>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c.<<RegisterWithDotnetSuggest>b__10_0>d.MoveNext()
--- End of stack trace from previous location ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass14_0.<<UseExceptionHandler>b__0>d.MoveNext()
```

from what I can see, it tries to read a solution folder `src/engines` as `engines` skipping the `src` folder.

**Comment by gregzhadko** [comment]
I have the same issue.

**Comment by Velorien** [comment]
Thanks for the heads up, this should be fixed in 0.9.3
